src/tests/pages/ActorPage.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { useAppSelector } from '../../store/store';
+import ActorPage from '../../pages/ActorPage/ActorPage';
+import renderWithProviders from '../../mockedData/test-utils';
+import dataActor from '../../mockedData/data-actor.json';
+import dataCards from '../../mockedData/data-movies.json';
+import dataImages from '../../mockedData/data-images.json';
+
+jest.mock('../../store/store', () => ({
+  __esModule: true,
+  ...jest.requireActual('../../store/store'),
+  useAppSelector: jest.fn(),
+}));
+
+describe('Actor page', () => {
+  beforeEach(() => {
+    (useAppSelector as jest.Mock).mockReturnValue({
```

Comment:
As I mentioned in a previous comment, you can use a function for returning a mock object.

(useAppSelector as jest.Mock).mockReturnValue(getUseAppSelectorMock());